src/main/resources/templates/timetracker/list.vm

```diff
@@ -44,12 +44,22 @@ $webResourceManager.requireResource("jira.webresources:ajax-preview", null)
                     <a href="$contextPath/browse/$worklog.issue" class="issue-link" title="#if($worklog.issueParent != '') $i18n.getText('plugin.parent.issue') $worklog.issueParent &#13 $worklog.issueSummary #else $worklog.issueSummary #end">$worklog.issue</a>
                   </td>
                   <td>
-                  #if($worklog.isClosed)
-                    <div class="tooltip-bottom"  title="$worklog.exactRemaining" style="color: #A3A1A1;">$worklog.roundedRemaining</div>
-                  #elseif($worklog.isMoreEstimatedTime)
-                    <div class="tooltip-bottom"  title="$worklog.exactRemaining">$worklog.roundedRemaining<div>
+                  #if($isRounded)
```

Comment:
Use the #if($isRounded) to set the div title and value. For this use #set($titleValue = $worklog.exactRemaining). Like this you don't have to copy 6 row.